src/parachain/loans.ts

```diff
@@ -335,6 +327,10 @@ export class DefaultLoansAPI implements LoansAPI {
         const currentBorrowIndex = Big(decodeFixedPointType(marketStatus[6]));
         const accumulatedDebt = this._calculateAccumulatedDebt(borrowedAmount, snapshotBorrowIndex, currentBorrowIndex);
 
+        if (borrowedAmount.eq(0)) {
+            return null;
+        }
```

Comment:
Suggestion: Move this up a bit to avoid calculating numbers that don't matter if we return `null`

Reply:
Thanks, that's very good catch, actually it also resolves bug caused by zero division that I found :raised_hands: